client/src/components/App.jsx

```diff
@@ -36,13 +42,18 @@ class App extends React.Component {
   };
 
   render() {
+    const { hostName, city, country, description, gettingAround, guidebookId, lat, lng } = this.state;
```

Comment:
not completely critical - but consider alphabetizing your restructured variables for easy lookup and cleaner style